go/kbfs/libdokan/folderlist.go

```diff
@@ -151,7 +151,9 @@ func (fl *FolderList) open(ctx context.Context, oc *openContext, path []string)
 				ctx, libkb.VLog1, "FL Lookup set alias: %q -> %q",
 				name, aliasTarget)
 			if !fl.isValidAliasTarget(ctx, aliasTarget) {
-				fl.fs.log.CDebugf(ctx, "FL Refusing alias to non-valid target %q", aliasTarget)
+				fl.fs.vlog.CLogf(
```

Comment:
should we also print a non-vdebug line without the filename here?

Reply:
Nah I think it should be fine.  The `reportErr` function will show the line number of the return, so an error message without any extra info wouldn't help in my opinion.